static/scripts/onboarding/onboarding.ts

```diff
@@ -227,9 +148,8 @@ const sodiumEncryptedSeal = async (publicKey: string, secret: string) => {
     const binsec = sodium.from_string(secret);
     const encBytes = sodium.crypto_box_seal(binsec, binkey);
     const output = sodium.to_base64(encBytes, sodium.base64_variants.URLSAFE_NO_PADDING);
-    defaultConf[KEY_NAME] = output;
-    defaultConf["evm-network-id"] = Number(chainIdSelect.value);
-    defaultConf["safe-address"] = safeAddressInput.value;
+    defaultConf[PRIVATE_ENCRYPTED_KEY_NAME] = output;
+    defaultConf[EVM_NETWORK_KEY_NAME] = Number(chainIdSelect.value);
```

Comment:
The [onboarding page](https://pay.ubq.fi/onboarding) has the "Safe address" text input. When config is generated the `safe-address` config param is missing. Could bring it back?

Reply:
Should be back now

Reply:
@FernandVEYRIER Sorry for the back and forth with the safe address param. Could you remove it completely along with the UI text input field?

Reply:
You good, I think it's very confusing for everyone due to all the different repos and versions of the config currently